### DISCUSSION TITLE
Ampliar o acesso é fundamental

### DISCUSSION THREAD

**Discussion by janaina2020** [james-sabel/tw-livre#6] [Ideias]
Será que nosso querido James Sabel toparia fazer um meetup na comunidade TW BRasil pra ensinar à galera como participar? Posso sugerir na próxima reunião de gestores?

**Comment by james-sabel** [comment]
Curti a ideia! Vamos conversando e, quando tivermos a data, lançamos na comunidade e aqui também.

@janaina2020 se vocês quiserem, podemos aproveitar essa thread aqui pra listar algumas dúvidas e sugerir tópicos que querem ver no dia do meetup. Que tal?
